src/borough/borough.service.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { BoroughModule } from "src/borough/borough.module";
+import { BoroughRepositoryMock } from "../../test/borough/borough.repository.mock";
+import { Test } from "@nestjs/testing";
+import { BoroughRepository } from "src/borough/borough.repository";
+import * as request from "supertest";
+import { BoroughController } from "src/borough/borough.controller";
+import { INestApplication } from "@nestjs/common";
+import { getBoroughsQueryResponseSchema } from "src/gen";
+
+describe("Borough service unit", () => {
+  let app: INestApplication;
+  let boroughController: BoroughController;
+
+  beforeEach(async () => {
+    const boroughRepositoryMock = new BoroughRepositoryMock();
+
+    const moduleRef = await Test.createTestingModule({
+      imports: [BoroughModule],
+    })
+      .overrideProvider(BoroughRepository)
+      .useValue(boroughRepositoryMock)
+      .compile();
+    app = moduleRef.createNestApplication();
+    await app.init();
+
+    boroughController = moduleRef.get<BoroughController>(BoroughController);
+  });
+
+  it("service should return a getBoroughsQueryResponseSchema compliant object", async () => {
```

Comment:
This is doing too much for a service unit test. It's also testing that the controller is formatting an http response. Please see this [example tax lot service test](https://github.com/NYCPlanning/ae-zoning-api/pull/148/files#diff-d7d6b01d6f364942bc836a4ce5a07545b238b9bddac1d4a9f4d23d2bb2cbe06cR14) for an example.